### PR TITLE
fix sources spec for HMMER 3.1b2 + minor style fixes + better sanity check

### PR DESCRIPTION
**Pull request by boegel**
fix for #3313

### CODE REVIEW

**Comment by fgeorgatos** [comment]
lgtm!

File: easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb [hunk 1]
```diff
@@ -28,16 +28,16 @@ description = """HMMER is used for searching sequence databases for homologs of
 toolchain = {'name': 'foss', 'version': '2016a'}
 
 source_urls = ['http://eddylab.org/software/hmmer%(version_major)s/%(version)s/']
-sources = ['hmmer-%(version)s-linux-intel-x86_64.tar.gz']
+sources = ['hmmer-%(version)s.tar.gz']
+
+runtest = 'check'
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb]: Is this were `runtest` is expected to go? Just curious, as I would tend to put it later.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2123.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/da2b56abf4f8acf85923e9a74b835331 for a full test report.

File: easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb]: Well, strictly speaking, order doesn't matter, but we tend to specify easyconfig parameters according to the steps they're used in'

`runtest` specifies how the included tests should be run (here with `make check`), which is done between the `build` and `install` steps, so I'm specifying it before `installopts` following the order of steps.

Comment (verdurin) [inline easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb]: Okay, thanks for the info.

Looks fine.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2441.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c1c568a5e299413f3b00228fd541c8ae for a full test report.

**Comment by boegel** [comment]
Thanks for the feedback @fgeorgatos and @verdurin!

**Pull request merged**